Sources/TuistCoreTesting/Models/TestAction+TestData.swift

```diff
@@ -12,7 +12,8 @@ public extension TestAction {
                      postActions: [ExecutionAction] = [],
                      diagnosticsOptions: Set<SchemeDiagnosticsOption> = Set(),
                      language: String? = nil,
-                     region: String? = nil) -> TestAction
+                     region: String? = nil,
+                     testPlans: [TestPlan] = []) -> TestAction
```

Comment:
To maintain the previous behaviour when test plans are not specified we'd need to default this to `nil` - I believe this why one of the unit tests was failing

Reply:
makes sense. changed.